### ISSUE TITLE
GLK Peer Review QA - Issue 17

### ISSUE THREAD

**Issue by gkerber** [Helen1479/FGHJ4567.github.io#20]
17.
Page Title: Flowers
Filename/URL:   https://helen1479.github.io/FGHJ4567.github.io/Flower.html
Type of Issue: Validity – Accessibility
Description of Issue:  Page failed accessibility testing:
**Accessibility Insights for Web** 
10 failures were detected:
color-contrast: Elements must have sufficient color contrast (8) WCAG 1.4.3
html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1
list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (1) WCAG 1.3.1
**Wave Web Accessibility Evaluation Tool**
5 contrast errors
